examples/training_scripts/lstm_wikitext2/lstm_wikitext2.py

```diff
@@ -22,7 +22,7 @@
 from syne_tune.report import Reporter
```

Comment:
Would this not move to benchmarking/training_scripts/ ?

Maybe the PR tool is showing this in a funny way ...